com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -960,5 +960,22 @@ public void Teleport(Vector3 newPosition, Quaternion newRotation, Vector3 newSca
             TryCommitValuesToServer(newPosition, newRotationEuler, newScale, m_CachedNetworkManager.LocalTime.Time);
             m_LocalAuthoritativeNetworkState.IsTeleportingNextFrame = false;
         }
+
+        /// <summary>
+        /// Override this and return false to follow the owner authoritative
+        /// Otherwise, it defaults to server authoritative
+        /// </summary>
+        protected virtual bool OnIsServerAuthoritatitive()
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// Determines if the <see cref="NetworkTransform"/> is server or owner authoritative.
+        /// </summary>
+        public bool IsServerAuthoritative()
+        {
+            return OnIsServerAuthoritatitive();
+        }
```

Comment:
I'm actually not entirely sure about exposing these things to public, especially from the SDK but it's fine I guess.

Reply:
I could make that internal...